file/filename.cc

```diff
@@ -364,7 +376,8 @@ Status SetCurrentFile(Env* env, const std::string& dbname,
   Slice contents = manifest;
   assert(contents.starts_with(dbname + "/"));
   contents.remove_prefix(dbname.size() + 1);
-  std::string tmp = TempFileName(dbname, descriptor_number);
+  std::string tmp =
+      TempFileName(dbname, descriptor_number) + "." + kEncryptionSkipName;
```

Comment:
Add a `kUnencryptedTempFileNameSuffix` and a method named `UnencryptedTempFileName`?

Reply:
`tmp.plain` sounds ok